Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN apt update && \
     curl -fsSL https://deb.nodesource.com/setup_16.x | bash - && \
     apt update && \
     apt install nodejs --yes
-RUN npm install -g yarn
+RUN npm install -g yarn --production
```

Comment:
This change is a noop. the `--production` flag only does something when it's installing from a package.json. You probably meant to add this to line 15 instead.

Reply:
Sorry ignore that previous comment, this is still a noop but if you move it to 15 then it won't have the necessary dependencies to actually build the project :)